tests/support/server.tcl

```diff
@@ -491,19 +493,25 @@ proc start_server {options {code undefined}} {
     set unixsocket [file normalize [format "%s/%s" [dict get $config "dir"] "socket"]]
     dict set config "unixsocket" $unixsocket
 
+    # Multiple modules can be passed in as overrides
+    set modules {}
+
     # apply overrides from global space and arguments
     foreach {directive arguments} [concat $::global_overrides $overrides] {
-        dict set config $directive $arguments
+        if {"loadmodule" eq $directive} {
+            set modules [concat $modules [list $arguments]]
```

Comment:
maybe we wanna have something slightly more generic? i.e. one that can also handle other repeated arguments like `user`, `save`, etc).
e.g. instead of a variable with module list, have a variable with plain lines to append, and we can put any config we like there.

Reply:
Ok, added `startup_arguments`